jax/experimental/compilation_cache/file_system_cache_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 Google LLC
```

Comment:
Another thing I just realized: this test file should actually go in [jax/tests](https://github.com/google/jax/tree/master/tests). This is where we put all test files, and this will make sure this file is included when we run all tests.

Reply:
If I move it to that directory it looks, it will not let me run this test file in the current directory where I have file_system_cache. Should I just make a copy or move it over at the end?

Reply:
You'll have to run the test from a different directory. I usually do everything from the base `jax` directory, so it'd be something like: `python3 tests/file_system_cache_test.py`

You'll also have to change the file_system_cache import on L16. I think `from jax.experimental.file_system_cache import FileSystemCache` should work, but you might have to experiment...